Facebook.iOS/source/AudienceNetwork/ApiDefinition.cs

```diff
@@ -85,6 +85,37 @@ interface AdChoicesView {
 		void UpdateFrameFromSuperview (UIRectCorner corner, UIEdgeInsets insets);
 	}
 
+	[Static]
+	partial interface AdExperienceType
+	{
+		// extern const FBAdExperienceType _Nonnull FBAdExperienceTypeRewarded;
+		[Field ("FBAdExperienceTypeRewarded", "__Internal")]
+		NSString Rewarded { get; }
+
+		// extern const FBAdExperienceType _Nonnull FBAdExperienceTypeInterstitial;
+		[Field ("FBAdExperienceTypeInterstitial", "__Internal")]
+		NSString Interstitial { get; }
+
+		// extern const FBAdExperienceType _Nonnull FBAdExperienceTypeRewardedInterstitial;
+		[Field ("FBAdExperienceTypeRewardedInterstitial", "__Internal")]
+		NSString RewardedInterstitial { get; }
+	}
+
+	// @interface FBAdExperienceConfig : NSObject
+	[BaseType(typeof(NSObject), Name= "FBAdExperienceConfig")]
+	[DisableDefaultCtor]
+	interface AdExperienceConfig
+	{
+		// @property (readwrite, nonatomic, strong) FBAdExperienceType _Nonnull adExperienceType;
+		[Export("adExperienceType", ArgumentSemantic.Strong)]
+		string AdExperienceType { get; set; }
+
+		// -(instancetype _Nonnull)initWithAdExperienceType:(FBAdExperienceType _Nonnull)adExperienceType __attribute__((objc_designated_initializer));
+		[Export("initWithAdExperienceType:")]
+		[DesignatedInitializer]
+		IntPtr Constructor(string adExperienceType);
```

Comment:
Since this seems to be expected to be one of the `AdExperienceType` constants the parameter needs to be NSString, this is because the underlying API could be doing pointer comparison instead of string contents comparison.

```suggestion
		IntPtr Constructor(NSString adExperienceType);
```

Reply:
Will take this one